src/extra/components/dld.py

```diff
@@ -169,32 +172,64 @@ def _build_reduced_pd(self, data, index, entry_level):
 
         return pd_cls(raw[finite_mask], pd.MultiIndex.from_frame(index_df))
 
-    def _insert_aligned_columns(self, df, columns):
+    @staticmethod
+    def insert_aligned_columns(df, columns):
```

Comment:
Is it a deliberate choice to make this method public? I think it will also appear in the docs, and I tend to think that keeping the documented API relatively small, so people can easily see the methods they want, is a good thing. But YMMV. :slightly_smiling_face: